anserini_dependency/api.py

```diff
@@ -109,7 +109,7 @@ def get_answers(question, num_hits, k):
     parser.add_argument('--no_cuda', action='store_false', help='do not use cuda', dest='cuda')
     parser.add_argument('--gpu', type=int, default=0) # Use -1 for CPU
     parser.add_argument('--seed', type=int, default=3435)
-    parser.add_argument('--dataset', help="the QA dataset folder {TrecQA|WikiQA}", default='../../data/TrecQA/')
+    parser.add_argument('--dataset', help="the QA dataset folder {TrecQA|WikiQA}", default='../../Castor-data/TrecQA/')
```

Comment:
two argument parsers in a file?

Reply:
Didn't understand this comment :(

Reply:
I see - that parser was already there (unchanged except for default path arg). This PR is only about the Castor-data issue. If that is a bug I would prefer to fix it separately in another PR.